src/progress_bar_exercise/components/button/Button.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import "./Button.scss";
+
+export default function Button({ children, onClick, color }) {
```

Comment:
Color is a loose word here for how it's used. It would be better to actually allow the Button component to handle a given color name, or rename it to 'variant' or 'style'